hbase_sql/src/test/java/com/splicemachine/derby/utils/VacuumIT.java

```diff
@@ -105,46 +100,77 @@
             .around(spliceTableIWatcher)
             .around(spliceTableJWatcher);
 
+    @Before
+    public void waitForStaleTransactions() throws Exception
+    {
+        // wait at max 60s for other transactions to finish
+        // we have defined SerialTest.class, so no other test should run in parallel
+        for(int i=0; i<60; i++) {
+            try (ResultSet rs = methodWatcher.executeQuery(
+                    "call SYSCS_UTIL.SYSCS_GET_ACTIVE_TRANSACTION_IDS()")) {
+                String actual = TestUtils.FormattedResult.ResultFactory.toString(rs);
+                if( actual.equals("") ) return;
+                Thread.sleep(1000);
+                System.out.println("VacuumIT: Waiting for active transactions:\n" + actual);
+            }
+        }
+        Assert.fail("VacuumIT can't run with active transactions still happening. This is a bug in other " +
```

Comment:
maybe we should just ignore the test instead of failing since `VacuumIT` itself didn't even start.